### PR TITLE
logentries, tile38: fix audit issue

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/tile38.rb [hunk 1]
```diff
@@ -31,6 +31,11 @@ def post_install
     datadir.mkpath
   end
 
+  def caveats; <<~EOS
+    To connect: tile38-cli
```
Comment (fxcoudert) [inline Formula/tile38.rb]: This does not seem specific to Homebrew, I recommend we simply drop the caveat.

Comment (ilovezfs) [reply Formula/tile38.rb]: The PR is just about fixing component order.

Comment (fxcoudert) [reply Formula/tile38.rb]: Sure, I am merely suggesting that we can improve it further and bring it in line with our policy :)

Comment (ilovezfs) [reply Formula/tile38.rb]: but https://github.com/Homebrew/homebrew-core/blob/master/Formula/mysql.rb#L148-L149

**Pull request merged**